internal/api/auth.go

```diff
@@ -96,6 +108,48 @@ func Signup(s storage.Store) http.HandlerFunc {
 	}
 }
 
+// Confirm ...
+func Confirm(s storage.Store) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		email := mux.Vars(r)["email"]
+		code := mux.Vars(r)["code"]
+		usr, err := s.Users().FindByEmail(email)
+		if err != nil {
```

Comment:
I can't understand what is `errs` Do you have a any document about this ? I want improve myself 😅

Reply:
Yeah, it is a little bit weird to send list of errors, I think, there is no particular documentation for it. Error handling section could be improved. Since, propagating errors can be appended as well, it can lead wrong debugging when error occurs. 
@yakuter know why it is implemented that way, I think, he will respond soon ;)

Reply:
The missing point in your approach, in my opinion, is that you are appending an error which is generated for particular case, however the `err` that you are checking against whether it is `nil` or not might have different error message which might be not related to `Email not found!`, in this case, debugger (-who try to solve the issue-) might have wrong starting point for solving issues like that.

Reply:
Oh I understand now. You are right. I think this is will be a other issue. Because this repository have a lot of them. And I will fix this now. Thank's for helpful review 😅

Reply:
@mrturkmencom can you check now. I added raw error message as a second element.

Reply:
yeah, it seems ok, for now, however, as I mentioned, error handling could be improved, but for now, it is ok 👍

Reply:
Well errs was a contribution for JWT tokens which I never liked. As you figured it out, we need a better error handling. Maybe with stack trace and different messages to users and developers. Let's discuss this later.